board/power_saving.h

```diff
@@ -1,4 +1,5 @@
 // WARNING: To stay in compliance with the SIL2 rules laid out in STM UM1840, we should never implement any of the available hardware low power modes.
+// Except while offroad (!)
```

Comment:
not relevant to the PR

Reply:
Discussed with George, added as a note for the future, also will add TODO comment there about power sleep modes.

Reply:
Will remove